rosbag2_cpp/src/rosbag2_cpp/service_utils.cpp

```diff
@@ -0,0 +1,136 @@
+// Copyright 2023 Sony Group Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <cstring>
+
+#include "rcl/service_introspection.h"
+
+#include "rosbag2_cpp/service_utils.hpp"
+#include "rosidl_typesupport_cpp/message_type_support_dispatch.hpp"
```

Comment:
```suggestion
#include "rosbag2_cpp/service_utils.hpp"
#include "rosidl_typesupport_cpp/message_type_support.hpp"
#include "rosidl_typesupport_cpp/message_type_support_dispatch.hpp"
```
Missing include, code doesn't compile without it.